index.html

```diff
@@ -13424,193 +13424,346 @@ <h2>IANA Considerations</h2>
-    -->
-    </pre>
-    <p>Requests the server to return the requested resource as JSON-LD
-      in both <a>compacted document form</a>
-      and <a>flattened document form</a>.
-      Note that as whitespace is used to separate the two URIs, they
-      are enclosed in double quotes (<code>"</code>).</p>
+    <p>This section will be submitted to the Internet Engineering Steering
+    Group (IESG) for review, approval, and registration with IANA.</p>
+
+    <p>The following section is based on the requirements defined in 
+      <a data-cite="rfc6838#section-6.2"></a>.</p>
+
+    <dl>
+      <!-- Full name of the well-defined structured syntax. -->
+      <dt>Name</dt>
+      <dd>JSON-based Serialization for Linked Data Structured Suffix</dd>
+      <!-- Suffix used to indicate conformance to the syntax. -->
+      <dt>+suffix</dt>
+      <dd>+ld+json</dd>
+
+      <!-- 
+      Include full citations for all specifications necessary to
+      understand the structured syntax.
+      -->
+      <dt>References</dt>
+      <dd>
+        <a href="https://w3c.github.io/json-ld-syntax/">
+          A JSON-based Serialization for Linked Data
+        </a>
+      </dd>
+
+      <!-- 
+      General guidance regarding encoding considerations for any type
+      employing this syntax should be given here.  The same requirements
+      for media type encoding considerations given in Section 4.8 apply
+      here.
+      -->
+      <dt>Encoding considerations</dt>
+      <dd>
+        See <a data-cite="RFC8259#section-11">RFC&nbsp;8259, section 11</a>.
+      </dd>
+
+      <!-- 
+      Any issues regarding the interoperable use of types employing this
+      structured syntax should be given here.  Examples would include
+      the existence of incompatible versions of the syntax, issues
+      combining certain charsets with the syntax, or incompatibilities
+      with other types or protocols.
+      -->
+      <dt>Interoperability considerations</dt>
+      <dd>
+        Not Applicable
+      </dd>
+
+      <!-- 
+      Generic processing of fragment identifiers for any type employing
+      this syntax should be described here.
+      -->
+      <dt>Fragment identifier considerations</dt>
+      <dd>
+        <p>
+          Fragment identifiers used with <a href="#structured-extension-ld-json">+ld+json</a>
+          are treated as in RDF syntaxes, as per
+          <a data-cite="RDF11-CONCEPTS#section-fragID">RDF 1.1 Concepts and Abstract Syntax</a>
+          [[RDF11-CONCEPTS]].
+        </p>
+      </dd>
+
+      <!-- 
+      Security considerations shared by media types employing this
+      structured syntax must be specified here.  The same requirements
+      for media type security considerations given in Section 4.6 apply
+      here, with the exception that the option of not assessing the
+      security considerations is not available for suffix registrations.
+      -->
+      <dt>Security considerations</dt>
```

Comment:
See comment above on referencing security considerations and moving content there.

Reply:
Same comment as above, you can push whatever you like to the branch.